OracleCoherence/README.md

```diff
@@ -120,13 +204,21 @@ Documentation covering the different aspects of running Oracle Coherence in Dock
 7. [JMX Monitoring](docs/5.monitoring) - Using JMX in Docker
 
 ## Issues
-If you find any issues with this Docker project, please report through the [GitHub Issues page](https://github.com/oracle/docker-images/issues).
+If you find any issues with this Docker project, please report through the
+[GitHub Issues page](https://github.com/oracle/docker-images/issues).
 
 ## Licenses
-To download and run Coherence Distribution regardless of inside or outside a Docker container, and regardless of which distribution, you must agree and accept the [OTN Standard License Terms](http://www.oracle.com/technetwork/licenses/standard-license-152015.html).
+To download and run Coherence Distribution regardless of inside or outside a Docker container,
+and regardless of which distribution, you must agree and accept the
+[OTN Standard License Terms](https://www.oracle.com/technetwork/licenses/standard-license-152015.html).
 
-To download and run Oracle JDK regardless of inside or outside a Docker container, you must agree and accept the [Oracle Binary Code License Agreement for Java SE](http://www.oracle.com/technetwork/java/javase/terms/license/index.html).
+To download and run Oracle JDK regardless of inside or outside a Docker container, you must agree
+and accept the [Oracle Binary Code License Agreement for Java SE](https://www.oracle.com/technetwork/java/javase/terms/license/index.html).
 
-All scripts and files hosted in this project on GitHub [docker-images/OracleCoherence](https://github.com/oracle/docker-images/OracleCoherence) repository required to build the Docker images are, unless otherwise noted, released under [UPL 1.0](https://oss.oracle.com/licenses/upl/), except for the files listed above with their specific licenses.
+All scripts and files hosted in this project on GitHub
+[docker-images/OracleCoherence](https://github.com/oracle/docker-images/OracleCoherence)
+repository required to build the Docker images are, unless otherwise noted, released under
+[UPL 1.0](https://oss.oracle.com/licenses/upl/), except for the files listed above with their
+specific licenses.
 
-Copyright (c) 2015, 2019, Oracle and/or its affiliates. All rights reserved.
+Copyright &copy; 2015, 2020, Oracle and/or its affiliates.
```

Comment:
Please don't use `&copy;` instead of (c). We need to keep this in human-readable plain ASCII text. The comma after 2020 is also not required.